src/epub/content.opf

```diff
@@ -36,8 +36,9 @@
 		<meta property="se:subject">Fantasy</meta>
 		<dc:description id="description">The people of Erl want magic, and get their wish.</dc:description>
 		<meta id="long-description" property="se:long-description" refines="#description">
-			&lt;p&gt;The people of the obscure village Erl demand to be ruled by a magic lord, so their ruler sends his son Alveric to Elfland to wed the elfin princess Lirazel. He brings her back to Erl, and the couple have a son. But Lirazel has trouble integrating with human society; and when a scheme by her father spirits her back and Elfland vanishes, Alveric begins a mad quest to find where Elfland went.&lt;/p&gt;
-			&lt;p&gt;&lt;i&gt;The King of Elfland’s Daughter&lt;/i&gt; is written in the pseudo-archaic prose style that &lt;a href="https://standardebooks.org/ebooks/lord-dunsany"&gt;Dunsany&lt;/a&gt; is known for, and that style can be polarizing. Some contemporaries thought the style didn’t work for a book of novel length; on the other hand, contemporary Irish writer George Russell called the book “the most purely beautiful thing Lord Dunsany has written.” The book touches on a range of themes, including the longing for fantastical things lost, the perception of time, sanity and madness, the fear of the unknown, and being careful what you wish for. Large passages are also devoted to hunting; the original edition even featured an illustration of a unicorn hunt opposite the title page. Neil Gaiman wrote an introduction to the 1999 edition, and Christopher Lee was a featured vocalist on a 1977 progressive rock album based on the book.&lt;/p&gt;
+			&lt;p&gt;The people of the obscure village Erl demand to be ruled by a magic lord, so their ruler sends his son Alveric to Elfland to wed the elfin princess Lirazel. He brings her back to Erl and the couple have a son, but Lirazel has trouble integrating with human society. When a scheme by her father spirits her away and Elfland vanishes, Alveric begins a mad quest to find where Elfland went.&lt;/p&gt;
+			&lt;p&gt;&lt;i&gt;The King of Elfland’s Daughter&lt;/i&gt; is written in the pseudo-archaic prose style for which &lt;a href="https://standardebooks.org/ebooks/lord-dunsany"&gt;Dunsany&lt;/a&gt;is known. Some contemporaries thought the style did not suit a novel-length work, but contemporary Irish writer George Russell called the book “the most purely beautiful thing Lord Dunsany has written.” Ursula K. Le Guin praised Dunsany’s style as “intensely mannered, intensely poetic,” and having “a delicate ear for speech-rhythms and a brilliantly exact imagination.” She called him “the most imitated, and the most inimitable, writer of fantasy,” adding “I have never seen any imitation Dunsany that consisted of anything beyond a lot of elaborate made-up names, some vague descriptions of gorgeous cities and unmentionable dooms, and a great many sentences beginning with ‘And.’ Dunsany is indeed the First Terrible Fate that Awaiteth Unwary Beginners in Fantasy.”&lt;/p&gt;
```

Comment:
I don't think we need all these quotes from UKLG. We want to avoid generic author bio material in the long description, because otherwise we run the risk of repeating ourselves for each of their books. Instead quotes should apply directly to the book in question (as Russell's does).